### PR TITLE
Update k8s to 1.20

### PR DESCRIPTION
**Pull request by yaron2**

### CODE REVIEW

File: charts/dapr/charts/dapr_rbac/templates/ClusterRoleBinding.yaml [hunk 1]
```diff
@@ -30,22 +30,22 @@ metadata:
   name: dapr-operator-admin
 rules:
 - apiGroups: ["*"]
-  resources: ["serviceaccounts", "deployments", "services", "configmaps", "secrets", "components", "configurations"]
+  resources: ["serviceaccounts", "deployments", "services", "configmaps", "secrets", "components", "configurations", "leases"]
```
Comment (artursouza) [inline charts/dapr/charts/dapr_rbac/templates/ClusterRoleBinding.yaml:33]: Why do we need "leases"?

**Review by artursouza — APPROVED** [review]

File: charts/dapr/charts/dapr_rbac/templates/ClusterRoleBinding.yaml (hunk 1 above)
Comment (yaron2) [reply charts/dapr/charts/dapr_rbac/templates/ClusterRoleBinding.yaml:33]: the kubebuilder framework used by the operator implements leader election for HA.

Comment (yaron2) [reply charts/dapr/charts/dapr_rbac/templates/ClusterRoleBinding.yaml:33]: In k8s > 1.18 there's an explicit need for RBAC permissions..

Comment (mukundansundar) [reply charts/dapr/charts/dapr_rbac/templates/ClusterRoleBinding.yaml:33]: can you add that as a comment in the yaml?

Comment (artursouza) [reply charts/dapr/charts/dapr_rbac/templates/ClusterRoleBinding.yaml:33]: I am OK with the yaml as-is.

**Pull request merged**